corebuild/integration/test/CommandRunner.cs

```diff
@@ -67,14 +67,15 @@ public int Run(out string commandOutput)
 				RedirectStandardOutput = true,
 				RedirectStandardError = true,
 			};
+                        outputHelper.WriteLine($"from caller working directory {Environment.CurrentDirectory}");
```

Comment:
Indentation is off. Also I would remove "from" and add ":" after directory for consistency with the message on line 78.

Reply:
Those spaces always seem to sneak in... thanks! Updating the message per your suggestion as well.